pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [project]
 name = "automata-lib"
-version = "8.1.0"
+version = "8.0.0"
```

Comment:
@eliotwrobson Ooh, I forgot to rebase `develop` from the latest `main` when I released v8.1.0 yesterday. Please allow me to fix. Then, we can revert this commit.

Reply:
@eliotwrobson Okay, `develop` has been rebased / fast-forwarded (06cdef7265751d3ba78704bac8a0b71f78cec4d9). Please revert this commit (38001475b99ad1c5d6d79ecc884b6c12d9461bfd) on the PR.

Reply:
Sounds good @caleb531. The paper changes themselves look fine to me, so I'll go ahead and approve so feel free to merge once this looks ready.